R/commonsummarystatsttestbayesian.R

```diff
@@ -341,15 +341,15 @@
   # Add BF10 or BF01 label for plots; never show the log Bayes factor in the plots (it interferes with the pie charts)
   if (options$bayesFactorType == "BF01")
     BFPlots <- "BF01"
-  else 
+  else
     BFPlots <- "BF10"
-  
+
 
   ttestPriorPosteriorPlot <- list(
     t        = t,
     n1       = n1,
     n2       = NULL,
-    paired   = TRUE,
+    paired   = isPairedOrOneSample,
```

Comment:
Where is this fed in? When is isPairedOrOneSample true and when is it false? I see that n2 is always NULL, but this means that this pertains only to the one-sample and paired-sample version. I remember the problem to be in the independent t-test.

Reply:
1) This is then used in a call to `jaspTTests::.plotPriorPosterior`. 
2) Regarding definition of `isPairedOrOneSample`, see https://github.com/jasp-stats/jaspSummaryStatistics/blob/6c14acd655831a7658333882e15855ad50f70db7/R/commonsummarystatsttestbayesian.R#L301-L312
3) You are right that `n2` is always null, but it would make sense if it's null only if it's paired/one-sample. It seems though that this does not affect the plot. I pushed a fix for this so that we always pass around the correct data in case the plotting functions will use it
4) Yes, this issue only manifested in the independent samples test, because the generic function that produces this results object always assumed paired/one-sample test, so the independent test was off.

Reply:
a little bit above there is
```r
  if(analysis == "oneSample" || analysis == "pairedSamples"){

    n2                  <- NULL
    ready               <- !(n1 == 0)
    isPairedOrOneSample <- TRUE

  } else if(analysis == "independentSamples"){
    n2                  <- options$n2Size
    ready               <- !(n1 == 0 || n2 == 0)
    isPairedOrOneSample <- FALSE
  }
```
so `n2` is not always `NULL`.

Reply:
So shouldn't `n2 = n2` in `ttestPriorPosteriorPlot`? Or am I missing something?

Reply:
> a little bit above there is ...

Yes, that's what I added now based on the first comment of @AlexanderLyNL 

> So shouldn't n2 = n2 in ttestPriorPosteriorPlot? Or am I missing something?

Yes, that's what I added in the latest commit

Reply:
Awesome everything seems fine to me then. I can't test this, as my developer module is still broken. Perhaps Don still has some comments.